xml/ha_maintenance.xml

```diff
@@ -0,0 +1,548 @@
+    </para>
+<screen>&prompt.root;systemctl stop pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     Reboot the node.
+    </para>
+   </step>
+  </procedure>
+
+  <para>
+   To check if the node joins the cluster again:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Log in to the node as &rootuser; or equivalent.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check if the &pace; service has started:
+    </para>
+<screen>&prompt.root;systemctl status pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     If not, start it:
+    </para>
+<screen>&prompt.root;systemctl start pacemaker.service</screen>
+   </step>
+   <step>
+    <para>
+     Check the cluster status with:
+    </para>
+<screen>&prompt.root;crm status</screen>
+    <para>
+     It should show the node coming online again.
+    </para>
+   </step>
+  </procedure>
+ </sect1>
+ <sect1 xml:id="sec.ha.maint.overview">
+  <title>Different Options for Maintenance Tasks</title>
+
+  <para>
+   &pace; offers a variety of options for performing system maintenance:
+  </para>
+
+  <variablelist>
+   <varlistentry xml:id="vle.ha.maint.mode.cluster">
+    <!--<term>Putting the Cluster Into Maintenance Mode</term>-->
+    <term><xref linkend="sec.ha.maint.mode.cluster" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      The global cluster property <literal>maintenance-mode</literal> allows
+      you to put all resources into unmanaged state at once. The cluster will
+      cease monitoring them and thus become oblivious to their status.
+     </para>
+     <note>
+      <title>Implications</title>
+      <para>
+       If the cluster is in maintenance mode, you can stop or restart cluster
+       resources at will&mdash;the &hasi; will not attempt to restart them. If
+       you stop the &pace; service on a node, all daemons and processes
+       (originally started as &pace;-managed cluster resources) will continue
+       to run.
+      </para>
+      <para>
+       If you attempt to start &pace; services on a node while the cluster is
+       in maintenance mode, &pace; will initiate a single one-shot monitor
+       operation (a <quote>probe</quote>) for every resource to evaluate which
+       resources are currently running on that node. However, it will take no
+       further action other than determining the resources' status.
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="vle.ha.maint.mode.node">
+    <!--<term>Putting a Node Into Maintenance Mode</term>-->
+    <term><xref linkend="sec.ha.maint.mode.node" xrefstyle="select:title"/></term>
+    <listitem>
+     <para>
+      Use this option if you want to do maintenance for plenty of (or all)
+      resources hosted on a node. It can also be used for updating software
+      packages on a node that do <emphasis>not</emphasis> affect the cluster
+      stack. See also <xref linkend="sec.ha.migration.update"/>.
```

Comment:
Hrm, I'm rethinking this... I actually can think of quite a few very different purposes of putting a node into maintenance node...
How about we just describe this similarly as previously in "cluster maintenance mode " like:
"allow you to put all the resources running on a node into maintenance state at once. The cluster will cease monitoring them and thus become oblivious to their status."